build/index.js

```diff
@@ -546,6 +547,11 @@ async function buildDocument(document, documentOptions = {}) {
   // raw HTML has been fixed to always have it in there already.
   injectNotecardOnWarnings($);
 
+  // Remove h4s from any existing notecards and transform them
+  // from <div class="note notecard"><h4>Note:</h4>foobar</div> to
+  // <div class="note notecard"><p><strong>Note:</strong>foobar</p></div>
```

Comment:
This comment belongs with the `format-notecard.js` instead.

Reply:
Got it!